src/main/cli/cli.ts

```diff
@@ -177,3 +162,24 @@ export async function spawnSubcommand(
     }
   );
 }
+
+export async function startRPC() {
+  await spawnSubcommand(
+    'start-daemon',
+    {
+      'server-address': `${RPC_BIND_HOST}:${RPC_BIND_PORT}`,
+    },
+    (logs) => {
+      store.dispatch(rpcAddLogs(logs));
+      RPC_LOG_EVENT_EMITTER.emit(logs);
+    },
+    (exitCode, exitSignal) => {
+      store.dispatch(rpcProcessExited({ exitCode, exitSignal }));
+      logger.error('RPC server has stopped');
+    },
+    (text) => {
+      store.dispatch(rpcAppendStdOut(text));
+    }
+  );
+  store.dispatch(rpcInitiate());
+}
```

Comment:
**Suggestion:** Consider adding error handling for the async function 'startRPC' at line 166. Currently, if an error occurs during the execution of 'spawnSubcommand', it will not be caught and could lead to unexpected behavior.
```suggestion
export async function startRPC() {
  try {
    await spawnSubcommand(...);
    ...
  } catch (e) {
    logger.error(`Error occurred: ${e.message}`);
  }
}
```